src/hearings/containers/request-hearing/hearing-check-answers/hearing-check-answers.component.ts

```diff
@@ -0,0 +1,29 @@
+import {Component, OnDestroy, OnInit} from '@angular/core';
+import {Store} from '@ngrx/store';
+import {ACTION} from '../../../models/hearings.enum';
+import {HearingsService} from '../../../services/hearings.service';
+import * as fromHearingStore from '../../../store';
+import {RequestHearingPageFlow} from '../request-hearing.page.flow';
+
+@Component({
+  selector: 'exui-hearing-check-answers',
+  templateUrl: './hearing-check-answers.component.html',
+})
+export class HearingCheckAnswersComponent extends RequestHearingPageFlow implements OnInit, OnDestroy {
+
+  constructor(protected readonly hearingStore: Store<fromHearingStore.State>,
+              protected readonly hearingsService: HearingsService) {
+    super(hearingStore, hearingsService);
+  }
+
+  public ngOnInit(): void {
```

Comment:
not needed

Reply:
I left this as this component is not finally built. But we will remove it if we don't use the ngOnInit method finally at later point.